paper.tex

```diff
@@ -112,6 +112,13 @@ \section{Motivation for central RSE units}
 Tasks that Research Software Engineers (RSEs) perform include training, \eg{} to improve the quality of code produced by researchers~\autocite{Ostlund2023}, consultation services, \eg{} regarding frameworks or algorithm selection, and the development of existing or new software.
 For an overview of typical tasks of RSEs and the competencies required, see~\autocite{goth_foundational_competencies_2024}, especially section\ 4.4:\ “RSE tasks and responsibilities”.
 
+Research software is getting increasingly complex.
+This is partly because the scientific problems being tackled using software
+are getting bigger.
+However, the software is also expected to be reliable, secure and easy to use.
+Handling this increased complexity whilst ensuring timely delivery requires
+dedicated RSEs.
```

Comment:
I agree with the idea here. But since this opens the paragraph on RSE UNITS, don't we want to include something for the UNIT aspect of it?

Reply:
Well, we did define dedicated RSEs as RSEs working in central units. Anyway, I have slightly expanded the sentence above to reiterate that.